src/components/EstimatedLifecycleCost/Review/index.tsx

```diff
@@ -30,64 +30,80 @@ const EstimatedLifecycleCostReview = ({
     year4: 'Year 4',
     year5: 'Year 5'
   };
-  const costForPhase = (items: LifecyclePhase[], phaseID: string) => {
-    const phaseMap: { [key: string]: string } = {
-      development: 'Development',
-      om: 'Operations and Maintenance',
-      other: 'Other'
-    };
-
-    const phase = items.find(obj => obj.phase === phaseMap[phaseID]);
-    if (phase) {
-      return parseFloat(phase.cost);
-    }
-    return undefined;
```

Comment:
This is where we were returning `undefined` for empty strings. When we do `parseFloat` on an empty string, it returns `NaN` which is technically of type `number`.